src/modules/72-triggers/pages/triggers/__tests__/SchedulePanel.test.tsx

```diff
@@ -125,7 +125,7 @@ function WrapperComponent(props: { initialValues: any }): JSX.Element {
         {formikProps => {
           return (
             <FormikForm>
-              <SchedulePanel {...defaultTriggerConfigDefaultProps} formikProps={formikProps} />
+              <SchedulePanel {...defaultTriggerConfigDefaultProps} formikProps={formikProps} isFixedUTCTime={true} />
```

Comment:
Nitpick, but you don't need to pass `true` for boolean props
```suggestion
              <SchedulePanel {...defaultTriggerConfigDefaultProps} formikProps={formikProps} isFixedUTCTime />
```

Reply:
that's true